providers/dns/scaleway/scaleway.go

```diff
@@ -95,12 +85,22 @@ func NewDNSProviderConfig(config *Config) (*DNSProvider, error) {
 		config.TTL = minTTL
 	}
 
-	client := internal.NewClient(internal.ClientOpts{
-		BaseURL: fmt.Sprintf("%s/domain/%s", config.BaseURL, config.Version),
-		Token:   config.Token,
-	}, config.HTTPClient)
+	configuration := []scw.ClientOption{
+		scw.WithAuth("SCWXXXXXXXXXXXXXXXXX", config.Token),
```

Comment:
It sounds weird to have a hardcoded access key (`SCWXXXXXXXXXXXXXXXXX`).
Is it expected?

Reply:
hi , to use the SDK an access key need to be add but it is not used ( you can directly use the API with a token only )
that why i use this hardcoded _dummy_ access key